.github/workflows/unit_tests.yml

```diff
@@ -109,6 +109,15 @@ jobs:
         run: py.test --cov=./ --cov-report=xml
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1
+        # Build and publish dbdocs
+      - name: Download dbdocs
+        run: npm install -g dbdocs
+      - name: Generate DBML
+        run: python manage.py dbml > cdh-web.dbml
+      - name: Login to dbdocs
+        run: echo ${{ secrets.DBDOCS_KEY }} | dbdocs login
+      - name: Push to dbdocs
+        run: dbdocs build cdh-web.dbml --project cdh-web
```

Comment:
Should this be part of unit tests?

I think a separate github action would make more sense; we don't need to regenerate these all the time. Suggest only run on push to `main`.